test/controllers/images_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ImagesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
let's add some meaningful tests here. We want to have a test for each action and each case, ie. a successful call to `create` and an unsuccessful request to `create`, a call to `show` when the image is found and a call to `show` when the image doesn't exist.